unpub/tool/pre_unpublish.dart

```diff
@@ -0,0 +1,49 @@
+import 'dart:io';
```

Comment:
2. This builds the flutter web project and copies the build files into unpub project as static strings.
```
--pwa-strategy=none to remove service worker and keep the build size small
--web-renderer=html to remove canvaskit and keep the build size small
--base-href=/delete/ as unpub_web is served at / so we need a subpath for serving the flutter app
```